### ISSUE TITLE
Program not creating data folder and / or config.ini 

### ISSUE THREAD

**Issue by tonydiep** [cabernetwork/cabernet#90]
Instructions say program will create data folder and config.ini file but error message says folder and / or file not found
```
Error Message:
FileNotFoundError: [Errno 2] No such file or directory: 'data/config.ini'
```

**Comment by rocky4546** [comment]
There is code to do this... Is there any exception when you run the app?

**Comment by rocky4546** [comment]
Try RC6...

**Comment by rocky4546** [comment]
Fixed in release 0.9.13.01
